public/sitemap.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+  <url loc="https://app.uniswap.org/" lastmod="2023-10-04T16:33:05.134Z" changefreq="daily" priority="0.7"/>
```

Comment:
should the priority of some of these pages be higher? i feel like /, /swap, /tokens, /nfts, /pools; in that order.